{{cookiecutter.project_slug}}/.github/workflows/pytest.yml

```diff
@@ -29,5 +29,7 @@ jobs:
           touch .env
           cat .env.example > .env
           cat .env
+      - run: mkdir -p client/dist/static
+      - run: pipenv run python server/manage.py collectstatic
```

Comment:
I don't love adding this, but we now have a test that expects a static image to exist.
I thought about mocking it in some way, but ultimately it's probably better to have it test with real things instead of mocked things.

Reply:
why not?

Reply:
something about needing to collectstatic in order to run unit tests feels wrong to me. In this case I think it's a good thing.